pkg/kwokctl/cmd/config/view/view.go

```diff
@@ -34,7 +34,7 @@ func NewCommand(ctx context.Context) *cobra.Command {
 	cmd := &cobra.Command{
 		Args:  cobra.NoArgs,
 		Use:   "view",
-		Short: "Display the default config file with --config",
+		Short: "Display the default config file",
```

Comment:
This is similar to the above, without changing the default configuration, just output it